frontend/src/features/map_planning/components/BaseStage.tsx

```diff
@@ -186,7 +186,35 @@ export const BaseStage = ({
   };
 
   return (
-    <div className="h-full w-full overflow-hidden">
+    <div
+      className="h-full w-full overflow-hidden"
+      onKeyDown={(e) => {
+        if (e.key === 'c' && e.ctrlKey) {
+          const nodes = trRef.current?.getNodes() || [];
+          if (nodes.length === 0) return;
+          dispatch({
+            type: 'OBJECT_COPY_TO_CLIPBOARD',
+            payload: nodes.map((o) => ({
+              id: o.id() + Math.random().toString(36).slice(2, 9), // TODO: How should we handle ids?
```

Comment:
I think at this point the id should already be known, so taking it directly from the node should be fine.
For the object creation case, there has to be another mechanism.
We could use UUIDs or rely on the backend for it.